src/pages/index.js

```diff
@@ -74,19 +33,63 @@ const validationOptions = {
 
```

Comment:
`validationOptions` should be in the `utils` folder 


From the 8th sprint all constants (including `initialCards` and `validationOptions`) should be in file `constants.js` in folder `utils` to clean up the main file.